### ISSUE TITLE
API-MS-WIN-SHCORE-TASKPOOL-L1-1-0

### ISSUE THREAD

**Issue by blueclouds8666** [Blaukovitch/API-MS-WIN_XP#2]
I'm looking for API-MS-WIN-SHCORE-TASKPOOL-L1-1-0.dll since i can't seem to find it anywhere. 
Forza Horizon 4 main executable requires this dll in order to launch, i'd like to solve this dependency so i can work on the other compatibility issues this game has and maybe, get it running on Windows 7 one day in conjunction with vkd3d.

This would only apply for Windows 7, but i thought i'd ask you nevertheless.

**Comment by Blaukovitch** [comment]
> I'm looking for API-MS-WIN-SHCORE-TASKPOOL-L1-1-0.dll since i can't seem to find it anywhere.

Hello!
Can you send the **main executable file** itself (PE COFF exe and dlls with) from Forza Horizon 4?
I believe that this is not the only library that is required to run the game. At the same time i look at the PE COFF import table.
I could not immediately find `API-MS-WIN-SHCORE-TASKPOOL-L1-1-0.dll` on Windows 10 22H2.

**Comment by blueclouds8666** [comment]
Yes, the game engine requires other "api-ms-win" dlls, but i managed to find them all, with the exception of this one.
It also has missing KERNEL32 and USER32 functions but those will be investigated and worked later on.

you can grab the engine binaries from here:
link will only be live for a short period. the password is "password":
https://icedrive.net/s/g91AivhQTbgj86ZWCCyQ4C1WybwW

**Comment by krimnassim** [comment]
@Blaukovitch hi,i can't find ur repository for win7 browsers crack it seems it got banned,where i can find it to download?

**Comment by adeii** [comment]
Did they forbid/block all your Chrome/Edge modds?

**Comment by Blaukovitch** [comment]
@krimnassim, @adeii, I've sent a request to the **GitHub** support service and am waiting for a response. **Unfortunately**, the reason why the **repository** was disabled was not even specified. If I get a reply, I will try to correct all claims and restore the repository again.

**Comment by Blaukovitch** [comment]
@blueclouds8666, overall, it really looks like a normal import table patch along the lines of the browser analogy.
However, there are also new WinAPIs, e.g. **CopyFile2**, **CreateFile2** that need to be added for Windows 7 as well.

**Comment by darkalexis1990** [comment]
@Blaukovitch why dont you try github alternatives? like for example discord or a forum like MDL ?

**Comment by Blaukovitch** [comment]
@darkalexis1990, Yes, I think you're right - I'll take that advice.

**Comment by darkalexis1990** [comment]
@Blaukovitch if you want my opinion, -> MDL forums. the best out there. :)

**Comment by adeii** [comment]
Welcome back, modded Chrome/iums!
